contents/docs/integrate/_snippets/install-js-snippet.mdx

```diff
@@ -27,7 +27,6 @@ Once the snippet is added, PostHog automatically captures `$pageview` and [other
 <details>
   <summary>Working with AI code editors?</summary>
 
-  If you’re working with AI code editors (like Lovable, Bolt.new, Replit, and others), it’s easy to install PostHog. All you have to do is ask (seriously). Specifically: in your tool, type a prompt like this: “I’d like to use PostHog to track events here; here is my install snippet.” Paste in your snippet. The tool will do the rest.
+  If you’re working with AI code editors (like Lovable, Bolt.new, Replit, and others), it’s easy to install PostHog. Just give it this prompt: `npx --yes @posthog/wizard@latest --signup`
```

Comment:
If we already have the AI wizard tab, we could just remove this entirely, I don't think it is necessary anymore.